test/sql/copy/parquet/test_parquet_remote.test

```diff
@@ -70,13 +70,13 @@ SELECT id, first_name, last_name, email FROM PARQUET_SCAN('https://github.com:44
 
 #
 #statement ok
-#SET s3_region='eu-central-1';
+#SET GLOBAL s3_region='eu-central-1';
```

Comment:
Can we change the HTTPFS/S3 reader so it follows the same code path as the `CURRENT_SETTING` function? (i.e. it will first check if there is a session variable, if there is not it will read the global variable)?

Reply:
I totally agree that client-scoped setting is ideal from the end user's perspective!
Can I address it in a separate PR though?

The problem is that currently the FileSystem instance is associated to the Database, not ClientContext.
So we have to pass down ClientContext-scoped data somehow (probably through OpenFile()).
So the change could be sizable and probably deserves some more discussion.
I'm happy to have a follow -up PR.

Reply:
Yes, that is fair. Perhaps we can set the default `SET` scope to global until that is fixed to keep the old behavior w.r.t. the S3 extension intact until the changes can be made there? That way we can avoid unnecessarily breaking other people's scripts/etc.

Reply:
Sounds like a plan! Updated.
Thanks for the suggestion!

Could you take another look?